views/mempool.tmpl

```diff
@@ -183,12 +184,13 @@
                                     {{template "decimalParts" (float64AsDecimalParts .TotalOut 8 false)}}
                                   </td>
                                   <td class="mono fs15 text-right">{{.Size}} B</td>
+                                  <td class="mono fs15 text-right">{{.FeeRate}}/kB</td>
```

Comment:
This gives `[ERR] EXPR: Template execute failure: template: mempool.tmpl:187:69: executing "mempool" at <.FeeRate>: can't evaluate field FeeRate in type types.MempoolTx`

Reply:
Fixed in the PR.